### PR TITLE
Explorer node deployment script +rj config

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: configs/benchmark-rongjian.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+  "description": "aws 464/600 nodes on 4 shard in 4 region, reserve 34 nodes for FN",
+  "client": {
+    "num_vm": 0,
```
Comment (alajko) [inline configs/benchmark-rongjian.json]: This is if you want to launch your own devnet? You just want launch your explorer nodes right.

File: pipeline/run_benchmark.sh [hunk 2]
```diff
@@ -123,6 +120,9 @@ function do_simple_cmd
       if [ "$CLIENT" == "true" ]; then
          CLIENT_JSON=',"role":"client"'
       fi
+
+      explorerArgs=benchmarkArgs
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: =$benchmarkArgs

File: pipeline/run_benchmark.sh [hunk 3]
```diff
@@ -200,9 +210,29 @@ EOT
       echo "$TIMEOUT -s SIGINT ${CURL_TIMEOUT} $CMD > $LOGDIR/$cmd/$cmd.$n.$ip.log" >> $cmd_file
    done
 
-   start_index=${configs[benchmark.shards]}
+# send commands to explorers at second
+
+   local explorer_shard_id=0
+   for n in $(seq ${configs[benchmark.shards]}+1 ${configs[benchmark.shards]}*2); do
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: +1, *2 won't work here

$(expr ${configs[benchmark.shards]} + 1)
$(...)

File: pipeline/run_benchmark.sh [hunk 4]
```diff
@@ -200,9 +210,29 @@ EOT
...
+      local ip=${NODEIPS[$n]}
+      CMD=$"curl -X GET -s http://$ip:1${PORT[$ip]}/$cmd -H \"Content-Type: application/json\""
+
+      case $cmd in
+         init|update|wallet)
+            sed "s/SHARDID/$explorer_shard_id/" $LOGDIR/$cmd/explorer.$cmd.json > $LOGDIR/$cmd/explorer.$cmd-$ip.json
+            CMD+=$" -d@$LOGDIR/$cmd/explorer.$cmd-$ip.json"
+            (( start_index ++ ))
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: no need.

File: pipeline/run_benchmark.sh [hunk 5]
```diff
@@ -200,9 +210,29 @@ EOT
...
+      esac
+
+      echo $n =\> $CMD
+      echo "$TIMEOUT -s SIGINT ${CURL_TIMEOUT} $CMD > $LOGDIR/$cmd/$cmd.$n.$ip.log" >> $cmd_file
+   done
+
+   start_index=${configs[benchmark.shards]}*2
    while [ $end -lt $NUM_NODES ]; do
-      start=$(( $PARALLEL * $group + ${configs[benchmark.shards]} + 1))
+      start=$(( $PARALLEL * $group + ${configs[benchmark.shards]}*2 + 1))
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: add space around *

File: pipeline/run_benchmark.sh (hunk 2 above)
Comment (rlan35) [reply pipeline/run_benchmark.sh]: Done.

File: pipeline/run_benchmark.sh (hunk 3 above)
Comment (rlan35) [reply pipeline/run_benchmark.sh]: Done

File: pipeline/run_benchmark.sh (hunk 4 above)
Comment (rlan35) [reply pipeline/run_benchmark.sh]: Done.

File: pipeline/run_benchmark.sh (hunk 5 above)
Comment (rlan35) [reply pipeline/run_benchmark.sh]: Done.

File: pipeline/run_benchmark.sh [hunk 6]
```diff
@@ -200,9 +210,29 @@ EOT
       echo "$TIMEOUT -s SIGINT ${CURL_TIMEOUT} $CMD > $LOGDIR/$cmd/$cmd.$n.$ip.log" >> $cmd_file
    done
 
-   start_index=${configs[benchmark.shards]}
+# send commands to explorers at second
+   local explorer_shard_id=0
+   local explorer_start_index=${configs[benchmark.shards]} + 1
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: this won't work, use $(( ${configs[benchmark.shards]} + 1 ))

File: pipeline/run_benchmark.sh [hunk 7]
```diff
@@ -200,9 +210,29 @@ EOT
...
+   local explorer_end_index=${configs[benchmark.shards]} * 2
```
Comment (LeoHChen) [inline pipeline/run_benchmark.sh]: same as above, use $(( ${configs[benchmark.shards]} * 2 ))

File: pipeline/run_benchmark.sh (hunk 5 above)
Comment (LeoHChen) [reply pipeline/run_benchmark.sh]: ?? still here.

Comment (harmony-ek) [reply pipeline/run_benchmark.sh]: Done. (2)

File: pipeline/run_benchmark.sh (hunk 6 above)
Comment (harmony-ek) [reply pipeline/run_benchmark.sh]: Fixed.

File: pipeline/run_benchmark.sh (hunk 7 above)
Comment (harmony-ek) [reply pipeline/run_benchmark.sh]: Fixed.

**Comment by harmony-ek** [comment]
> LGTM, please merge it after your test.

Yep, did and things work correctly (explorer has an issue of its own though, @lzl124631x should be working on it now).

**Pull request merged**